docs/source/reference/CLI.md

```
@@ -0,0 +1,329 @@
+
+```bash
+zipline compile
+zipline compile --chronon-root /path/to/project
+```
+
+## zipline run
+
+Runs Spark-based jobs for self-hosted deployments. This is the primary command for executing data pipelines on your own infrastructure.
+
+```bash
+zipline run [OPTIONS]
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)